### ISSUE TITLE
off table armed offer roof short route merit human lazy blanket stereo bench fame gallery staff mistake add huge note height menu hint audit

### ISSUE THREAD

**Issue by Gcr99** [ethereum-lists/chains#5402]

**Comment by Gcr99** [comment]
Tuu

**Comment by 777easy** [comment]
Yes uh what is it's the meaning for Why are you saying This email Like
don't make no sense Issue five number five 5402
shane Lassere

**Comment by 777easy** [comment]
They're not getting me no clarity with this uh a lot these emails me I
don't know what they really mean like Um I know I'm having a lot of trouble
I thought I was the last couple last week I thought I was getting
everything in the control but seems like I'm not because all my emails are
coming seven days late So I thought I had was getting everything going
everything was going smooth but I guess it's not f*** I'm gonna try and so
hard so I could get the trading but I don't want to trade when I'm losing
everything That'd be stupid on my part Right to keep on adding to the
f****** ish add into the far When I can't put it out so I'm looking at
putting the fire out before I go Do anything That's why ain't trading this
sucks because That's my that's hard to do I want to do it so bad but I've
been trying but I can't get Access to my assets because I just b******* so
Thanks for putting up with all those b******* and hopefully it's upgrading
y'all Um But if y'all need anything from me y'all just let me know I am
working man a tire salary they're trying to figure this out But For some
reason I thought I had it but I didn't The only thing that's really helping
me is what the downloads I get from y'all like on great GitHub So like
that's the only thing like that that's the debug and all that s*** that
really does help some Because before I they was Random just pushing this
s*** up my way making me do what they want So now they just hauling my
emails or something I don't know what's going on but thank you all I don't
know what this means what the issue is but I'm gonna go check it out But I
noticed every time I go check it out or something like that and I say
something I don't get no feedback so like I don't get nobody title on me
nothing back so I don't know what's going on So somebody taking my emails
or looking at all this s*** reading it before I'm reading it All right so
that's the reason because I gotta read to figure out what I got to do so
they read it so they can figure out they do before me
shane Lassere
